Problem3.cpp

```diff
@@ -0,0 +1,38 @@
+#include<iostream>
+using namespace std;
+
+
+//binarySearch(int arr[],int left,int right);
+
+
+int binarySearch(int arr[], int left, int right)    //Binary search implemented.
+{
+
+    if (left<=right) {
+        int mid = left + (right - left) / 2;
+
+
+        if (arr[mid]>arr[mid+1]&& arr[mid]>arr[mid-1]) {		// Check if Middle is greater than both left and right side.
```

Comment:
not a correct way to approach this problem take some testcases then you will be able to get the point